docs/application/dotnet/guides/nlp/nlp.md

```diff
@@ -0,0 +1,252 @@
+# Natural Language Processing (NLP)
+
+
+Nlp is a subset of natural language toolkit that specifies an interface and a protocol for basic natural language processing. Tizen enables you to use Natural Language Process (Nlp) functionalities, such as language detection, parts of speech, word tokenize, lemmatize, and named entity detection. For more information, see the [nltk Forum](http://www.nltk.org/).
+
+The main features of the Tizen.Nlp namespace include:
+
+-   Word Tokenize suppport
+
+    You can get tokens from a sentence, the type of return is [WordTokenizeResult](#wordtokenize). 
+    This method breaks up the sentence into words and punctuation.
+
+-   Part of Speech support
+
+    You can get tokens and tags from a sentence, the type of return is [PosTagResult](#postag). This method breaks up the sentence into words and punctuation with tags attributes.
+
+-   Named Entity Detection support
+
+    You can get tokens and tags from a sentence, the type of return is [NamedEntityRecognitionResult](#nechunk). This method breaks up the sentence into words and punctuation with tags attributes.
+
+-   Lemmatizing support
+
+    You can get the actual word from a word, the type of return is [LemmatizeResult](#lemmatize). This method is used to get the actual word from a word.
+
+-   Language Detect support
+
+    You can get language from a sentence, the type of return is [LanguageDetectedResult](#langdetect). This method is used to detect the language of a sentence.
+
+
+## Prerequisites
+
+Enable your application to use the Nlp functionality:
```

Comment:
@divyk 
When you check other pages, you can see there is a kind of format in Prerequisites section.

```
To enable your application to use the blahblah functionality:

1. To use the~~

2. To use the~~
```
Isn't it better to get the consistency with other pages?
Please refer other Prerequisites sections and give your opinion.